public_html/app/models/Post.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+    class Post {
```

Comment:
This is model. So you should create Abstract Model class and extend every model with it.
AbstractModel should share function between all models.
get()
add()
delete()
..etc..

Reply:
1. Do i understand correctly? These functions add() and so on should be dynamical an to be able to take variables, so that in the ( ) i would select which column to select and the row to delete? Something like that?

Reply:
No. https://www.php.net/manual/en/language.oop5.abstract.php . You should create abstraction layer for models.